draft-pauly-quic-datagram.md

```diff
@@ -182,29 +170,11 @@ and can store the contents in memory.
 
 DATAGRAM frames MUST be protected with either 0-RTT or 1-RTT keys.
 
-## Flow Identifiers {#flow-id}
-
-Flow identifiers represent bidirectional flows of datagrams within a single QUIC
-connection. These are effectively equivalent to UDP ports, that allow basic
-demultiplexing of application data. Whenever one side of a connection
-receives a frame with a Flow ID was was not previously known, it MAY represent
-this to the application as a new flow of datagrams.
-
-The primary role of the QUIC transport towards the flow identifier is to provide
-a standard mechanism for demultiplexing application data flows, which may be
-destined for different processing threads in the application, akin to UDP sockets.
-
-Beyond this, a sender SHOULD ensure that DATAGRAM frames within a single flow
-are transmitted in order relative to one another. If multiple DATAGRAM frames can
-packed into a single packet, the sender SHOULD group them by Flow ID to
-promote fate-sharing within a specific flow and improve the ability to process batches
-of datagram messages efficiently on the receiver.
-
-Applications that do not have a need for the Flow ID can use the value zero on
-their DATAGRAM frames and clear the FLOW_ID bit to omit sending the identifier
-over the wire. If an application uses a mixture of DATAGRAM frames with and
-without the FLOW_ID bit set, the frames without it are assumed to be part of the
-application-level flow with Flow ID zero.
+Application protocols using datagrams might need to differentiate categories or
+flows of datagrams being transmitted over a single QUIC connection.
+Each application protocol is expected to define its mechanism for
```

Comment:
Each application protocol is expected to define its "own" mechanism ...

Reply:
Fixed! Thanks.